demo/sync/syncGlobalManager.js

```diff
@@ -20,6 +20,10 @@ function initSync() {
         return reject(error);
       }
       var cloudUrl = $fh.getCloudURL();
+
+      // decode the cloud app url if its encoded - required for keycloak
+      cloudUrl = decodeURIComponent(cloudUrl);
```

Comment:
This is a feature in keycloak to escape data provided in URL's when providing values eg `?url=` , so i don't think we should removing this. 

$fh.getCloudUrl seems to be the issue here actually. Its just grabbing the redirected URL format. This should come from a config file instead in a URL un-encoded format.